apps/web/src/pages/get-started/consts.tsx

```diff
@@ -0,0 +1,53 @@
+import { ProductUseCasesEnum } from '@novu/shared';
+import { RingingBell } from '@novu/design-system/dist/cjs';
+
+export const onboardingUseCases: OnboardingUseCases = {
+  [ProductUseCasesEnum.IN_APP]: {
+    steps: ['Configure In-App provider', 'Build a workflow', 'Connect trigger and run test', 'Track activity feed'],
```

Comment:
Please keep in mind that these steps might become more complex, as they are tied with the descriptions, and also descriptions have links, etc.

Reply:
Yes i am aware, other then link we should not have other complexity at the moment.
small note for digest i will probably wont reuse this const as we would need to make some custom code there, but for the simple steps with just text and link we could use reuse same component.